web/index.html

```diff
@@ -18,7 +18,10 @@
               aria-expanded="false" aria-label="Toggle navigation">
         <span class="navbar-toggler-icon"></span>
       </button>
-      <a id="nav-home" class="navbar-brand" data-gerbera-menu-cmd="HOME" title="Gerbera"><img src="assets/theme/gerbera-logo-white.png" class="d-inline-block align-top" alt="Gerbera Logo"></a>
+      <a id="nav-home" class="navbar-brand" data-gerbera-menu-cmd="HOME" title="Gerbera" href="#">
```

Comment:
We do not need the href because it's handled by jquery.

Reply:
An `a` element without the href will not get the pointer styling on hover, plus it does no harm.